docs/guides/pagination/connections.md

```diff
@@ -526,30 +531,39 @@ class User:
         description="The age of the user."
     )
 
+    @staticmethod
+    def from_row(row: Dict[str, Any]):
+        return User(
+            id=row['id'],
+            name=row['name'],
+            occupation=row['occupation'],
+            age=row['age']
+        )
+
 
 @strawberry.type
 class Query:
     @strawberry.field(description="Get a list of users.")
     def get_users(self, first: int = 2, after: Optional[str] = None) -> Connection[User]:
         if after is not None:
           # decode the user ID from the given cursor.
-          user_id = decode_user_cursor(cursor=after)
+          user_id = decode_user_cursor(cursor=after) - 1
```

Comment:
why do we need this -1 here? 😊

Reply:
Without it it was failing to show the values of the next page.